src/visualizer/svg.js

```diff
@@ -15,26 +17,49 @@ const {
     const svgLineSpace = svgWidth * 0.04; // the amount of space between each line, by the width of the svg
     const svgLineWidth = svgWidth * 0.01;
     const svgTextSize = svgLineSpace * 0.5;
-    const RANGEWIDTH = 22;
+    const LSRANGEWIDTH = 22; // Learningstyle rangewith, from -11 to 11 = 22
+    const WAHRANGEWIDTH = 2; // Working At Home rangewith, from -1 to 1 = 2
 
     /**
      * @summary Creates and appends a div element with the svg of the learningstyles
      * @param {object} group the group array with students
```

Comment:
Det er vel bare en group og ikke en group array

Reply:
* @param {object} group the group array with students
->
 * @param {object} group the group object with students